config/ecs/ecs-amp-xray.yaml

```diff
@@ -0,0 +1,54 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+  awsecscontainermetrics:
+
+processors:
```

Comment:
From what I understand there is no need to rename ECS infra metrics when exporting to AMP like we do in the EMF exporter

Reply:
Also AMP reported metrics won't have ECS labels until https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/4624